ethstats/ethstats.go

```diff
@@ -423,18 +426,18 @@ func (s *Service) login(conn *connWrapper, sendCh chan *StatsPayload) error {
 	if info := infos.Protocols[istanbul.ProtocolName]; info != nil {
 		ethInfo, ok := info.(*eth.NodeInfo)
 		if !ok {
-			return errors.New("Could not resolve NodeInfo")
+			return errors.New("could not resolve NodeInfo")
 		}
 		network = fmt.Sprintf("%d", ethInfo.Network)
 		protocol = fmt.Sprintf("%s/%d", istanbul.ProtocolName, istanbul.ProtocolVersions[0])
 	} else {
 		lesProtocol, ok := infos.Protocols["les"]
 		if !ok {
-			return errors.New("No less protocol found")
+			return errors.New("no less protocol found")
```

Comment:
```suggestion
			return errors.New("no LES protocol found")
```
(while we're touching this line, may as well fix the "less" and capitalize it